uint32  id             # id сообщения 
uint32 pinMotorEn      # Стутус пина управления драйвером моторов, включен драйвер или нет
Struct_Motor[4] motor     # Структура мотора